### DISCUSSION TITLE
Skip db:create?

### DISCUSSION THREAD

**Discussion by samip5** [mastodon/mastodon#19648] [Q&A] [answered]
How can I skip the db:create one? I have the database already, and I just want it to push the schema etc to it?

Not using Docker db, and not a root user on the database server.

**Comment by Gargron** [comment, marked as answer]
Yes, you can. The commands are:

- `db:create` - Create (empty) database
- `db:schema:load` - Create tables and indexes in the database
- `db:seed` - Insert some basic records into the database that Mastodon relies on
- `db:setup` - The 3 commands above in one
- `db:migrate` - Upgrade existing database schema to newer version

If you run the interactive `mastodon:setup`, it runs `db:setup` in the background.
